File: cpp/home_work_1/task2.cpp
#include <iostream>

int main() {
    int x,y,z, min;
    std::cout << "Please Enter X, Y, Z" << std::endl;
    std::cin >> x >> y >> z;

    min = x;

    if (y < min) {
        min = y;
    }

    if (z < min) {
        min = z;
    }

    std::cout << "Min number is " << min << std::endl;

    return 0;
}

File: java/home_work_1/task2.java
import java.util.*;
class Task1 {
    public static void main(String[] args) {
        
            int x,y,z, min;
            System.out.println("Please Enter X, Y, Z" + "\n");
            Scanner sx = new Scanner(System.in);
            Scanner sy = new Scanner(System.in);
            Scanner sz = new Scanner(System.in);
            x = sx.nextInt();
            y = sy.nextInt();
            z = sz.nextInt();
            
            min = x;
    
            if (y < min) {
                min = y;
            }
    
            if (z < min) {
                min = z;
            }

    System.out.println("min number is " + min) ;
    }
}
